api/src/main/java/me/lokka30/treasury/api/economy/response/EconomyFailureReason.java

```diff
@@ -40,13 +48,15 @@
          */
         @Override
         public @NotNull String getDescription() {
-            return "That feature is currently not available during migration!";
+            return "That feature is currently not available during migration.";
         }
     },
 
     /**
      * A constant representing failure due to request cancellation.
      *
+     * //TODO document further. check what this constant would be used for.
```

Comment:
that todo should not be placed inside here as it will go to the html form of the javadocs. move it above the javadoc comment.